src/Build/Resources/xlf/Strings.es.xlf

```diff
@@ -432,7 +432,7 @@
       <trans-unit id="UnhandledMSBuildError">
         <source>This is an unhandled exception in MSBuild -- PLEASE UPVOTE AN EXISTING ISSUE OR FILE A NEW ONE AT https://aka.ms/msbuild/unhandled.
     {0}</source>
-        <target state="new">This is an unhandled exception in MSBuild -- PLEASE UPVOTE AN EXISTING ISSUE OR FILE A NEW ONE AT https://aka.ms/msbuild/unhandled.
+        <target state="translated">Esta es una excepción no controlada en MSBuild. VOTE POR UN PROBLEMA EXISTENTE O PRESENTE UNO NUEVO EN https://aka.ms/msbuild/unhandled.
```

Comment:
```suggestion
        <target state="translated">Ésta es una excepción no controlada por MSBuild. VOTE POR UN PROBLEMA EXISTENTE O PRESENTE UNO NUEVO EN https://aka.ms/msbuild/unhandled.
```
?

Reply:
> If there are issues in translations, visit https://aka.ms/ceLocBug and log bugs for fixes.

The translators don't use GitHub and won't see comments here.

Reply:
That felt like too much for tiny grammar things, but I still feel like I should comment if I see things 🙂